build/CoreData/dll/CoreData.vcxproj

```diff
@@ -25,6 +25,15 @@
     <None Include="CoreData.def" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="..\..\CoreFoundation\dll\CoreFoundation.vcxproj">
+      <Project>{81F30AF6-EAC3-4DFA-929A-C25D69E8080B}</Project>
+    </ProjectReference>
+    <ProjectReference Include="..\..\Foundation\dll\Foundation.vcxproj">
+      <Project>{86127226-9A6E-439B-A070-420A572AF0C7}</Project>
+    </ProjectReference>
+    <ProjectReference Include="..\..\Starboard\dll\Starboard.vcxproj">
+      <Project>{0AC27ECF-E2AB-420B-9359-4843FFF4CBFA}</Project>
+    </ProjectReference>
```

Comment:
Good catch, thanks.

---
In reply to: [70720366](https://github.com/Microsoft/WinObjC/pull/610#discussion_r70720366) [](ancestors = 70720366)